charts/orchestrator/templates/_helpers.tpl

```diff
@@ -120,4 +120,25 @@
         {{- end -}}
         {{- .argoCDNamespace -}}
     {{- end -}}
+{{- end -}}
+
+{{- define "get-cluster-version" -}}
+  # Retrieve OCP version from `clusterversion` object
+  {{- $version :=(lookup "config.openshift.io/v1" "ClusterVersion" "" "version") }}
+  {{- range $version.status.history }}
+    {{- if eq .state "Completed" }}
+      {{- $v:=(semver .version) }}
+      {{- printf "OCP version is %d.%d" $v.Major $v.Minor }}
```

Comment:
Remember to remove this line unless you want to use it for debugging, in which case perhaps better to add it to the NOTES.txt.

Reply:
I'll remove.